### PR TITLE
Update issues workflow

### LINKED ISSUE #571: Issues workflow is outdated
**Description**

[Issues workflow](https://github.com/kyma-project/community/blob/main/governance/03-issues-workflow.md) is outdated. We don't use ZenHub and teams sprints Github milestones anymore. 

**Assignees**

@kyma-project/technical-writers

### PR DESCRIPTION
**Pull request by Sawthis**
**Description**

Changes proposed in this pull request:

- Remove Zenhub mentions
- Remove not used milestones

**Related issue(s)**

Resolves #571

### CODE REVIEW

File: governance/03-issues-workflow.md [hunk 1]
```diff
@@ -57,37 +57,27 @@ There are eight different stages of the triage:
...
 
->**NOTE**: Issues are taken from the main Kyma backlog by different teams that are responsible for specific areas of Kyma. This is the actual workflow in Kyma which allows distinguishing which team works on a specific issue. This approach allows easy work in team sprints using ZenHub board. We are aware that our teams' names may be cryptic for the external community, but at the moment we are not able to provide any better solution. If you have a better idea, your feedback would be highly appreciated. Sorry for the inconvenience.
+>**NOTE**: Issues are taken from the main Kyma backlog by different teams that are responsible for specific areas of Kyma. This is the actual workflow in Kyma which allows distinguishing which team works on a specific issue. This approach allows easy work in team sprints using Github Issues. We are aware that our teams' names may be cryptic for the external community, but at the moment we are not able to provide any better solution. If you have a better idea, your feedback would be highly appreciated. Sorry for the inconvenience.
```
Comment (Disper) [inline governance/03-issues-workflow.md:62]: Could this note be removed as well? 
As I understand, we don't have any team mentions at the moment. We're using area labels, and assigning people to the issues instead.

**Review by Disper — APPROVED** [review]
One smaller comment, other than that, LGTM

File: governance/03-issues-workflow.md (hunk 1 above)
Comment (Sawthis) [reply governance/03-issues-workflow.md]: done

**Review by Disper — APPROVED** [review]

**Review by mmitoraj — CHANGES_REQUESTED** [review]
I added two comments.

I noticed one additional piece to correct. In line 32 could you and the word "to" after the word "related". The line should like that:
```* `area/{CAPABILITY_NAME}` (#3CB913) indicates which capabilities are related to a given issue. You can assign more than one `area` label to an issue.```

**Review by mmitoraj — APPROVED** [review]

**Pull request merged**